dev/start.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+set -ex
+
+function clean {
+echo 'Removing test environment'
+echo '---'
+docker-compose down -v
+rm -rf inventory.tmp
+}
+
+# normalises project name by filtering non alphanumeric characters and transforming to lowercase
+declare -x COMPOSE_PROJECT_NAME
+
+COMPOSE_PROJECT_NAME=$(echo "${BUILD_TAG:-ansible-plugin-testing}-conjur-host-identity" | sed -e 's/[^[:alnum:]]//g' | tr '[:upper:]' '[:lower:]')
+echo "${COMPOSE_PROJECT_NAME}" > compose_project_name
+
+declare -x ANSIBLE_CONJUR_AUTHN_API_KEY=''
+declare -x CLI_CONJUR_AUTHN_API_KEY=''
+declare cli_cid=''
+declare conjur_cid=''
+declare ansible_cid=''
+
+# get conjur client auth api key
+function api_key_for {
+  local role_id=$1
+  if [ -n "$role_id" ]
+  then
+  docker exec "${conjur_cid}" rails r "print Credentials['${role_id}'].api_key"
+  else
+  echo ERROR: api_key_for called with no argument 1>&2
+  exit 1
+  fi
+  }
```

Comment:
This file needs some re-formatting to adhere to our [Bash style guide](https://github.com/cyberark/community/blob/main/Conjur/conventions/bash-style-guide.md#bash-style-guide-and-best-practices). For example:
```suggestion
# get conjur client auth api key
function api_key_for {
  local role_id=$1
  if [ -n "$role_id" ]; then
    docker exec "${conjur_cid}" rails r "print Credentials['${role_id}'].api_key"
  else
    echo ERROR: api_key_for called with no argument 1>&2
    exit 1
  fi
}
```

Reply:
Done.